packages/app/src/pages/landelijk/vaccinaties.tsx

```diff
@@ -327,3 +403,36 @@ const VaccinationPage: FCWithLayout<typeof getStaticProps> = ({
 VaccinationPage.getLayout = getNationalLayout;
 
 export default VaccinationPage;
+
+const TooltipList = styled.ol`
+  margin: 0;
+  padding: 0;
+  list-style: 'none';
```

Comment:
Shouldn't it be without the quotes, since you are in the interpolation way of passing styles?

```suggestion
  list-style: none;
```

Reply:
Somehow this works too though :)